lib/evmone/baseline.cpp

```diff
@@ -398,12 +401,16 @@ evmc_result execute(const VM& vm, ExecutionState& state, const CodeAnalysis& ana
         }
 
         case OP_JUMP:
-            pc = op_jump(state, analysis.jumpdest_map);
+            pc = op_jump(state, analysis.jumpdest_map, analysis.padded_code.get());
+            if (state.status == EVMC_BAD_JUMP_DESTINATION)
```

Comment:
This is quite ugly, without it if `op_jump` jumps to final `OP_INVALID`, returned status will be overwritten witn `EVMC_INVALID_INSTRUCTION`.

Reply:
It could be instead a check for `pc` value:
```suggestion
            if (pc ==  analysis.padded_code.get() + header.code_end(state.code.size()))
```

Reply:
This was refactored on master.